concrete/single_pages/dashboard/reports/logs.php

```diff
@@ -27,9 +27,33 @@
         <div class="ccm-search-field-content">
             <?=$form->select('channel', $channels)?>
             <?php if (isset($selectedChannel)) { ?>
-            <a href="<?=$controller->action('clear', $valt->generate(), $selectedChannel)?>" class="btn btn-default btn-danger pull-right" style="margin-top: 30px;"><?=tc('%s is a channel', 'Clear all in %s', Log::getChannelDisplayName($selectedChannel))?></a>
+            <a href="javascript:void(0)" class="btn btn-default btn-danger pull-right" onclick="clearSelectedChannelLogs()" style="margin-top: 30px;"><?=tc('%s is a channel', 'Clear all in %s', Log::getChannelDisplayName($selectedChannel))?></a>
+            <script>
+                clearSelectedChannelLogs = function() {
+                    ConcreteAlert.confirm(
+                        '<?=t('Are you sure you want to clear the %s channel logs?', Log::getChannelDisplayName($selectedChannel)); ?>',
```

Comment:
I'd write
```javascript
ConcreteAlert.confirm(
    <?= json_encode(t('Are you sure ...')) ?>,
   ..
);
```

That way, even if translations contain an apostrophe, we won't have any problem.